docs/Components-Screen.md

```diff
@@ -86,6 +149,29 @@ The valid values for this prop are: `"handled"`, `"always"`,and `"never"`.
 
 ```tsx
 <Screen preset="scroll" keyboardShouldPersistTaps="never">
+  <Header headerTitle="screen" />
+  // ... content here ...
+</Screen>
+```
+
+### `ScrollViewProps`
+
+The `ScrollViewProps` prop is a `ScrollViewProps` object that is passed to the React Native [`ScrollView`](https://facebook.github.io/react-native/docs/scrollview.html) component. This only applies for the `scroll` preset.
+
+```tsx
+<Screen preset="scroll" ScrollViewProps={{ contentContainerStyle: { padding: 10 } }}>
+  <Header headerTitle="screen" />
+  // ... content here ...
+</Screen>
+```
+
+### `scrollEnabledToggleThreshold`
+
+The `scrollEnabledToggleThreshold` prop is an optional `number` that determines the threshold at which the `scrollEnabled` prop of the `ScrollView` is toggled. This only applies for the `auto` preset. The default value is `{ percent: 0.92 }`. You can pass a point value in lieu of a percentage, e.g. `{ point: 100 }` will enable scrolling when the scroll view height is greater than 100 points larger than the scroll view content height.
```

Comment:
I think this should be "when the scroll view height is less than 100 points larger than the scroll view content height". Greather than/less than is hard, I swear.